webcomponents/gn-search-input/src/app/gn-search-input.component.ts

```diff
@@ -13,7 +18,21 @@ import {
   encapsulation: ViewEncapsulation.ShadowDom,
 })
 export class GnSearchInputComponent implements OnInit {
+  @Input() apiUrl = '/'
+  @Input() primaryColor = '#9a9a9a'
```

Comment:
We might need an abstract class if we want to add this to all components.
Also, maybe add doc in web component README

Reply:
+1 for abstract class